core/src/main/java/hudson/tasks/ArtifactArchiver.java

```diff
@@ -245,15 +276,15 @@ public void perform(Run<?,?> build, FilePath ws, Launcher launcher, TaskListener
                     listenerWarnOrError(listener, Messages.ArtifactArchiver_NoMatchFound(artifacts));
                     String msg = null;
                     try {
-                    	msg = ws.validateAntFileMask(artifacts, FilePath.VALIDATE_ANT_FILE_MASK_BOUND, caseSensitive);
+                        msg = ws.validateAntFileMask(artifacts, FilePath.VALIDATE_ANT_FILE_MASK_BOUND, caseSensitive);
```

Comment:
Arrrrrrgh. Is this really worth worrying about? These weren't indented by the standard depth in the first place...